lib/board.rb

```diff
@@ -0,0 +1,56 @@
+require 'pry'
+
+class Board
+attr_accessor :cells, :token, :player
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def reset!
+    cells.clear
+    initialize
+  end
+
+  def input_to_index(user_input)
+    user_input.to_i - 1
+  end
+
+  def position(user_input)
+    index = input_to_index(user_input)
+    self.cells[index]
+  end
+
+  def full?
+    !self.cells.include? " "
+  end
+
+  def turn_count
+    cells.count { |token| token == "X" || token == "O" }
+  end
+
+  def taken?(user_input)
+    if position(user_input) == " "
```

Comment:
You can use the implicit return of a boolean expression rather than returning true or false explicitly. 
```ruby
def taken?(input)
    !(position(input) == " " || position(input) == "")
end
```